vm-mock/vm.js

```diff
@@ -376,6 +379,53 @@ export default function createMockedABI(
       assembly_script_get_call_coins() {
         return BigInt(0);
       },
+
+      assembly_script_transfer_coins(_addressPtr, _coinsAmount) {
+        const address = ptrToString(_addressPtr);
+        if (!ledger.has(address)) {
+          ledger.set(address, {
+            storage: new Map(),
+            contract: '',
+            balance: 0,
+          });
+        }
+        ledger.get(address).balance += Number(_coinsAmount);
```

Comment:
i think balances should be in BigInt, isn't it ?

Reply:
you're right! Done